packages/devtools/src/getMonolithicCSSRules.ts

```diff
@@ -1,16 +1,28 @@
 import * as stylis from 'stylis';
 import type { AtomicRules, MonolithicAtRules, MonolithicRules, RuleDetail } from './types';
 
+const griffelClassNameRegex = /^\.[0-9a-z]+$/;
```

Comment:
Well, it's not enterily correct regex as all classes always start with `f`. Can you make it more accurate?